themes/default/content/docs/intro/console/auditing.md

```diff
@@ -172,6 +172,7 @@ This is a list of the audit log events currently being recorded.
 | Policy Pack Deleted                      | indicates the deletion of a policy pack                                                                              |
 | Policy Pack Disabled                     | indicates the disabling of a policy pack                                                                             |
 | Policy Pack Enabled                      | indicates the enabling of a policy pack                                                                              |
+| Secret Decrypted                         | indicates the decryption of a secret value associated with the specified stack                                                                  |
```

Comment:
looks like other items in this table use a mix of `to a stack` and  `for a stack`, we should probably align with one of those instead of introducing a new format

Reply:
Not sure I follow what you mean here? The other items discuss adding/removing something to/from the stack. Since secrets are technically a separate entity, doesn't seem right to say "indicates the decryption of a secret value from a stack". Can you suggest a wording?

Reply:
We could do something like "indicates the decryption of a secret value associated with a stack"

Reply:
```suggestion
| Secret Decrypted                         | indicates the decryption of a secret value associated with a stack                                                                  |
```

I like that - @susanev WDYT?

Added it as a suggestion so Sean can make the change quickly if we're in agreement.

Reply:
oh i was not trying to block, I approved when I left the comment. thx for the suggestion dave!